packages/kbn-apm-config-loader/package.json

```diff
@@ -6,9 +6,10 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "build": "../../node_modules/.bin/tsc",
+    "build": "yarn clean && ../../node_modules/.bin/tsc",
     "kbn:bootstrap": "yarn build",
-    "kbn:watch": "yarn build --watch"
+    "kbn:watch": "yarn build --watch",
+    "clean": "../../node_modules/.bin/del target"
```

Comment:
Why can't we just use `rm -rf`?

Reply:
We were using a mixture - decided to go with using `del` for now. We require a linux emulator for POSIX API's like `rm` for Windows development, so it's probably a mute point. Do you see any reason to prefer `rm` over the `del` package? When cleaning the target in code, we are also using the `del` package.

Reply:
In the two contexts I think there is a different expectation for what things will look like. `rm -rf` communicates very efficiently what it's doing to most people familiar with scripting and `../../node_modules.bin/del` communicates something too but it's not nearly as efficient I'd say.

Reply:
It's also very constrained space, the relative path feels unnecessary and messy.

Reply:
Done in c487267a0a84d93e33fe335d6f3f3d679754b4ea